### PR TITLE
C17 12 rename step column enum

### PR DESCRIPTION
**Pull request by salammohamadi**

### CODE REVIEW

File: src/constants/tables/inputOptions.ts [hunk 1]
```diff
@@ -94,24 +94,24 @@ export const operators: MappedOption[] = [
...
+  { value: WRSteps.DATA_CONTROL, label: 'DataControl', style: { direction: 'ltr' } },
+  { value: WRSteps.RE_REQUESTER, label: 'Requester Approvement', style: { direction: 'ltr' } },
+  { value: WRSteps.FINISHED, label: 'Finished', style: { direction: 'ltr' } },
 ];
 
 export const equipmentStatus:MappedOption[] = [
-  { label: 'در حال کار', value: 1, extraClassName: 'running' },
-  { label: 'متوقف', value: 2, extraClassName: 'stopped' },
-  { label: 'برنامه ریزی شده', value: 3, extraClassName: 'planned' },
+  { label: 'در حال کار', value: 1, style: { color: '#0FC76E' } },
```
Comment (da-saman) [inline src/constants/tables/inputOptions.ts:112]: add this colors to colorNames

Comment (salammohamadi) [reply src/constants/tables/inputOptions.ts]: done

File: src/constants/colorNames.ts [hunk 2]
```diff
@@ -17,3 +17,7 @@ export const ORANGE_1 = '#FDB900';
 export const ORANGE_2 = '#F7D883';
 
 export const RED_1 = '#EA0000';
+
+export const RUNNING = '#0FC76E';
```
Comment (da-saman) [inline src/constants/colorNames.ts:21]: Running is color name? please provide names like green, red ...

Comment (salammohamadi) [reply src/constants/colorNames.ts]: done

**Review by da-saman — APPROVED** [review]

**Pull request merged**